README.md

```diff
@@ -12,16 +12,17 @@ These are the steps how to do releases for the Kolibri Windows installer.  Note
 
 We are releasing `v1.3.1` as an example here.
 
-1. Tag the `master` branch as `v1.3.1` at `kolibri-installer-windows`
+1. Set the new version in KolibriSetupScript.iss where we set the KOLIBRI_INSTALLER_VERSION env var.
+2. Tag the `develop` branch as `v1.3.1` at `kolibri-installer-windows`
```

Comment:
if we are tagging `develop`, we might want to rename it to `main` and delete `master`